spec/support/pages/projects/index.rb

```diff
@@ -122,9 +122,12 @@ def expect_no_project_create_button
         expect(page).to have_no_css('[data-test-selector="project-new-button"]')
       end
 
-      def expect_gantt_button(disabled: false)
-        expect(page).to have_css("button#{disabled ? '[disabled]' : ''}",
-                                 text: 'Open as Gantt view')
+      def expect_gantt_menu_entry(visible: false)
+        if visible
+          expect(page).to have_css('#projects-index-open-as-gantt', text: 'Open as Gantt view')
+        else
+          expect(page).to have_no_css('#projects-index-open-as-gantt', text: 'Open as Gantt view')
+        end
```

Comment:
You can use `have_(no)_link` and not rely on the id anymore :) 
```suggestion
      def expect_gantt_menu_entry(visible: false)
        if visible
          expect(page).to have_link('Open as Gantt view')
        else
          expect(page).to have_no_link('Open as Gantt view')
        end
```